frontend/src/components/StatusTransfer.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {useEffect} from "react";
+import {
+    useContractRead
+} from "@starknet-react/core";
+import {TREASURY_ADDRESS} from "../constants/config.json";
+import {TreasuryABI} from "../lib/treasuryABI";
+import CancelTransferBtn from "./CancelTransferBtn";
+import {formatBalance} from "../lib/erc20";
+
+const StatusTransfer = () => {
+
+    const getLiveTransfer = useContractRead({
+        functionName: 'get_live_transfers',
+        address: TREASURY_ADDRESS,
+        abi: TreasuryABI,
+    })
+
+    useEffect(() => {
+        getLiveTransfer.refetch()
+    },[])
```

Comment:
use `const { data, isLoading }`

No reason to have the useEffect

And then this data should be shown in the table – refer to e.g. Proposals.tsx for how it's done elsewhere.